.github/workflows/ci-app-examples.yml

```diff
@@ -13,7 +13,6 @@ on:
       - "tests/tests_app_examples/**"
       - "examples/app_*"
       - "requirements/app/*"
-      - "!requirements/app/docs.txt"
```

Comment:
I had to remove these because checkgroup does not support exclusion like this at the moment.

And then, if this file was modified, checkgroup would require that this workflow runs but this is configured to not do it.